packages/cli/src/commands/new/csharp.ts

```diff
@@ -72,10 +72,15 @@ export function CSharp(): CommandModule<{}, Args> {
       const project = new Project(projectDir, name, 'csharp').addTemplate(template);
 
       if (ttk) {
-        project.addTeamsToolkit();
+        project.addTeamsToolkit('csharp');
```

Comment:
@aacebo is this safe for TeaksToolkit Attributes?

Reply:
this code isn't used yet cause the c# sdk isn't done, but the only configuration available to TTK currently is 'basic'